iml-agent/src/action_plugins/lustre/snapshot.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{agent_error::ImlAgentError, lustre::lctl};
+use structopt::StructOpt;
+
+#[derive(serde::Deserialize, Debug, StructOpt)]
+pub struct Create {
+    /// Filesystem name
+    fsname: String,
+    /// Snapshot name
+    name: String,
+
+    /// Optional comment for the snapshot
+    #[structopt(short = "c", long = "comment")]
+    comment: Option<String>,
+}
```

Comment:
This is going to have to move to `wire-types` with a compile flag for `StructOpt` support.

Reply:
That's what I thought about the output, but apparently, input will be wired as well.

Reply:
Input will be sent from the manager in Rust. Since the input is structured, it makes sense to share it.

Reply:
Done.